src/index.ts

```diff
@@ -1,10 +1,28 @@
 import { Box, Quaternion as CQuaternion, ConvexPolyhedron, Cylinder, Shape, Sphere, Trimesh, Vec3 } from 'cannon-es';
 import { Box3, BufferGeometry, CylinderGeometry, MathUtils, Mesh, Object3D, SphereGeometry, Vector3 } from 'three';
 import { ConvexHull } from '../lib/ConvexHull.js';
-import { getComponent, getGeometry, getVertices } from './utils.js';
+import { getComponent, getGeometry, getVertices } from './utils';
 
 const PI_2 = Math.PI / 2;
 
+export type BoxParameters = [x: number, y: number, z: number];
+
+export type CylinderParameters = [radiusTop: number, radiusBottom: number, height: number, segements: number];
+
+export type SphereParameters = number;
+
+export type ConvexPolyhedronParameters = { vertices: [number, number, number][], faces: number[][] };
```

Comment:
The inconsistency of the parameter types might be a little confusing for users —

| type | structure |
|------|-----------|
| SphereParameters | `number` |
| ConvexPolyhedronParameters | `{vertices, faces}` |
| TrimeshParameters | `[vertices, faces]` |

I think the object syntax is probably my favorite of these what do you think of this? Haven't cross-referenced with Cannon.js constructors, not sure if we strictly want to follow those or keep it a bit more open-ended.

| type | structure |
|------|-----------|
| SphereParameters | `{radius: number}` |
| ConvexPolyhedronParameters | `{vertices, faces}` |
| TrimeshParameters | `{vertices, faces}` |

Reply:
I agree - I think I was over-indexing on matching the cannon constructor params exactly. I'll change the types to all use the object syntax, that sounds good!